lib/node_modules/@stdlib/blas/ext/base/dsum/README.md

```diff
@@ -75,15 +73,12 @@ Note that indexing is relative to the first index. To introduce an offset, use [
 
 ```javascript
 var Float64Array = require( '@stdlib/array/float64' );
-var floor = require( '@stdlib/math/base/special/floor' );
 
 var x0 = new Float64Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
 var x1 = new Float64Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
 
-var N = floor( x0.length / 2 );
-
-var v = dsum( N, x1, 2 );
-// returns 5.0
+var v = dsum( 4, x1, 2 );
+// returns 1.0
```

Comment:
```suggestion
// returns 5.0
```

Reply:
Hi, was just checking the error log, thanks for the change!